src/preferences/pages/blacklist.ui

```diff
@@ -3,11 +3,22 @@
   <requires lib="gtk" version="4.0" />
   <requires lib="adw" version="1.0" />
   <template class="PrefsBlacklist" parent="AdwPreferencesPage">
-    <property name="title" translatable="yes">Blacklist</property>
+    <property name="title" translatable="yes">Blacklist / Whitelist</property>
     <property name="icon-name">action-unavailable-symbolic</property>
+    <child>
+      <object class="AdwPreferencesGroup">
+        <property name="title" translatable="no">Whitelist</property>
+        <child>
+          <object class="AdwSwitchRow" id="useWhitelist">
+            <property name="title" translatable="no">Use as Whitelist</property>
+            <property name="subtitle" translatable="no">When selected, instead of ommiting certain windows, only allows certain windows.</property>
```

Comment:
```suggestion
        <property name="title" translatable="yes">Whitelist</property>
        <child>
          <object class="AdwSwitchRow" id="useWhitelist">
            <property name="title" translatable="yes">Use as Whitelist</property>
            <property name="subtitle" translatable="yes">When selected, instead of ommiting certain windows, only allows certain windows.</property>
```